packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -176,12 +208,12 @@ function Install(options) {
 
             if (!url && packetName !== 'example') {
                 if (options.stopDb) {
-                    if (objects && objects.destroy) {
-                        await objects.destroy();
+                    if (this.objects && this.objects.destroy) {
```

Comment:
```suggestion
                    if (this.objects?.destroy) {
```

Reply:
etc :)

Reply:
Code-wise this.states and this.objects should always be there. So probably it is currently just unneeded defensive code